auditlog/README.rst

```diff
@@ -1,29 +1,42 @@
-Track user operation on data models
-===================================
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=================================
+Audit Log - Track user operations
+=================================
 
 This module allows the administrator to log user operations performed on data
 models such as ``create``, ``read``, ``write`` and ``delete``.
 
 Usage
 =====
 
-Go to `Reporting / Audit / Rules` to subscribe rules. A rule defines which
+Go to `Settings / Technical / Audit / Rules` to subscribe rules. A rule defines which
 operations to log for a given data model.
-Then, check logs in the `Reporting / Audit / Logs` menu.
 
-During installation, it will migrate any existing data from the `audittrail`
-module (rules and logs).
+Then, check logs in the `Settings / Technical / Audit / Logs` menu. You can group them by
+user sessions, date, data model or HTTP requests:
+
+Get the details:
+
+.. image:: /auditlog/static/description/log.png
```

Comment:
I set all these image URLs in my last PR, but it seems they do not work properly on apps.odoo.com (https://apps.odoo.com/apps/modules/8.0/auditlog/). We should change them to relative ones like `static/description/IMAGE.png` in order to have them displayed.